sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorTraceExporter.cs

```diff
@@ -17,6 +17,8 @@ public class AzureMonitorTraceExporter : BaseExporter<Activity>
         private readonly AzureMonitorExporterOptions _options;
         private readonly string _instrumentationKey;
         private readonly ResourceParser _resourceParser;
+        private readonly StorageTransmissionEvaluator _storageTransmissionEvaluator;
+        private const int SampleSize = 10;
```

Comment:
nit: Consider a different name or add a comment, this variable name is too generic.